### ISSUE TITLE
Error installing in Windows

### ISSUE THREAD

**Issue by billdenney** [nlmixrdevelopment/n1qn1c#1]
Related to nlmixrdevelopment/RxODE#278.

When trying installation on Windows, I get an error:

```
> devtools::install_github("nlmixrdevelopment/n1qn1c")
Downloading GitHub repo nlmixrdevelopment/n1qn1c@HEAD
√  checking for file 'C:\Users\Bill Denney\AppData\Local\Temp\RtmpacNW62\remotes1738532347bc\nlmixrdevelopment-n1qn1c-265ee3b/DESCRIPTION'
-  preparing 'n1qn1':
√  checking DESCRIPTION meta-information ... 
-  cleaning src
-  running 'cleanup.win'
-  checking for LF line-endings in source and make files and shell scripts
-  checking for empty or unneeded directories
-  building 'n1qn1_6.0.1-9.tar.gz'
   Warning: file 'n1qn1/cleanup' did not have execute permissions: corrected
   
Installing package into ‘C:/Users/Bill Denney/Documents/R/win-library/4.0’
(as ‘lib’ is unspecified)
* installing *source* package 'n1qn1' ...
** using staged installation
** libs

*** arch - i386
"C:/rtools40/mingw32/bin/"g++ -std=gnu++11  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -mfpmath=sse -msse2 -mstackrealign -c RcppExpMod.cpp -o RcppExpMod.o
"C:/rtools40/mingw32/bin/"gcc  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c ajour.c -o ajour.o
"C:/rtools40/mingw32/bin/"gcc  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c calmaj.c -o calmaj.o
"C:/rtools40/mingw32/bin/"gcc  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c init.c -o init.o
"C:/rtools40/mingw32/bin/"gcc  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c n1qn1_all.c -o n1qn1_all.o
"C:/rtools40/mingw32/bin/"gcc  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c proj.c -o proj.o
"C:/rtools40/mingw32/bin/"gcc  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c satur.c -o satur.o
C:/rtools40/mingw32/bin/g++ -std=gnu++11 -shared -s -static-libgcc -o n1qn1.dll tmp.def RcppExpMod.o ajour.o calmaj.o init.o n1qn1_all.o proj.o satur.o -LC:/PROGRA~1/R/R-40~1.2/bin/i386 -lR
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_]+0x288): undefined reference to `dgemm_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_]+0x416): undefined reference to `dgemv_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb1ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_[__ZN4arma10glue_times5applyIdLb0ELb1ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_]+0x302): undefined reference to `dgemm_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb1ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_[__ZN4arma10glue_times5applyIdLb0ELb1ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_]+0xb8e): undefined reference to `dgemv_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb1ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_[__ZN4arma10glue_times5applyIdLb0ELb1ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_]+0xf07): undefined reference to `dsyrk_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_]+0x25a): undefined reference to `dgemm_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_]+0x52f): undefined reference to `dgemm_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_]+0x841): undefined reference to `dgemv_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_]+0x926): undefined reference to `dgemv_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_]+0xae8): undefined reference to `dgemv_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_]+0xbcd): undefined reference to `dgemv_'
collect2.exe: error: ld returned 1 exit status
no DLL was created
ERROR: compilation failed for package 'n1qn1'
* removing 'C:/Users/Bill Denney/Documents/R/win-library/4.0/n1qn1'
* restoring previous 'C:/Users/Bill Denney/Documents/R/win-library/4.0/n1qn1'
Error: Failed to install 'n1qn1' from GitHub:
  (converted from warning) installation of package ‘C:/Users/BILLDE~1/AppData/Local/Temp/RtmpacNW62/file173840625726/n1qn1_6.0.1-9.tar.gz’ had non-zero exit status
> 
```

**Comment by billdenney** [comment]
Looking more, this appears to be an issue that BLAS is not linked.  I tried a few things locally, but I couldn't get it working.

**Comment by mattfidler** [comment]
See if this works.

**Comment by billdenney** [comment]
I still have the issue:

```
devtools::install_github("nlmixrdevelopment/n1qn1c")
Downloading GitHub repo nlmixrdevelopment/n1qn1c@HEAD
√  checking for file 'C:\Users\Bill Denney\AppData\Local\Temp\Rtmp2nqj7r\remotes44a44ad46106\nlmixrdevelopment-n1qn1c-c359213/DESCRIPTION'
-  preparing 'n1qn1':
√  checking DESCRIPTION meta-information ... 
-  cleaning src
-  running 'cleanup.win'
-  checking for LF line-endings in source and make files and shell scripts
-  checking for empty or unneeded directories
-  building 'n1qn1_6.0.1-9.tar.gz'
   Warning: file 'n1qn1/cleanup' did not have execute permissions: corrected
   
Installing package into ‘C:/Users/Bill Denney/Documents/R/win-library/4.0’
(as ‘lib’ is unspecified)
* installing *source* package 'n1qn1' ...
** using staged installation
** libs

*** arch - i386
"C:/rtools40/mingw32/bin/"g++ -std=gnu++11  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -mfpmath=sse -msse2 -mstackrealign -c RcppExpMod.cpp -o RcppExpMod.o
"C:/rtools40/mingw32/bin/"gcc  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c ajour.c -o ajour.o
"C:/rtools40/mingw32/bin/"gcc  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c calmaj.c -o calmaj.o
"C:/rtools40/mingw32/bin/"gcc  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c init.c -o init.o
"C:/rtools40/mingw32/bin/"gcc  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c n1qn1_all.c -o n1qn1_all.o
"C:/rtools40/mingw32/bin/"gcc  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c proj.c -o proj.o
"C:/rtools40/mingw32/bin/"gcc  -I"C:/PROGRA~1/R/R-40~1.2/include" -DNDEBUG  -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/RcppArmadillo/include' -I'C:/Users/Bill Denney/Documents/R/win-library/4.0/Rcpp/include'        -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c satur.c -o satur.o
C:/rtools40/mingw32/bin/g++ -std=gnu++11 -shared -s -static-libgcc -o n1qn1.dll tmp.def RcppExpMod.o ajour.o calmaj.o init.o n1qn1_all.o proj.o satur.o -LC:/PROGRA~1/R/R-40~1.2/bin/i386 -lR
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_]+0x288): undefined reference to `dgemm_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_]+0x416): undefined reference to `dgemv_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb1ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_[__ZN4arma10glue_times5applyIdLb0ELb1ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_]+0x302): undefined reference to `dgemm_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb1ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_[__ZN4arma10glue_times5applyIdLb0ELb1ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_]+0xb8e): undefined reference to `dgemv_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb1ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_[__ZN4arma10glue_times5applyIdLb0ELb1ELb0ENS_3MatIdEES3_EEvRNS2_IT_EERKT3_RKT4_S4_]+0xf07): undefined reference to `dsyrk_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_]+0x25a): undefined reference to `dgemm_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_]+0x52f): undefined reference to `dgemm_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_]+0x841): undefined reference to `dgemv_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_]+0x926): undefined reference to `dgemv_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_]+0xae8): undefined reference to `dgemv_'
C:/rtools40/mingw32/bin/../lib/gcc/i686-w64-mingw32/8.3.0/../../../../i686-w64-mingw32/bin/ld.exe: RcppExpMod.o:RcppExpMod.cpp:(.text$_ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_[__ZN4arma10glue_times5applyIdLb0ELb0ELb1ELb0ENS_3MatIdEES3_S3_EEvRNS2_IT_EERKT4_RKT5_RKT6_S4_]+0xbcd): undefined reference to `dgemv_'
collect2.exe: error: ld returned 1 exit status
no DLL was created
ERROR: compilation failed for package 'n1qn1'
* removing 'C:/Users/Bill Denney/Documents/R/win-library/4.0/n1qn1'
* restoring previous 'C:/Users/Bill Denney/Documents/R/win-library/4.0/n1qn1'
Error: Failed to install 'n1qn1' from GitHub:
  (converted from warning) installation of package ‘C:/Users/BILLDE~1/AppData/Local/Temp/Rtmp2nqj7r/file44a476be6549/n1qn1_6.0.1-9.tar.gz’ had non-zero exit status
> 
```

**Comment by billdenney** [comment]
I confirmed that `Rblas.dll` and `Rlapack.dll` are in "C:/PROGRA~1/R/R-40~1.2/bin/i386" which appears to be the library directory linked.

Does it need `-lblas` and `-llapack` or something like that on the `g++` line?

**Comment by billdenney** [comment]
I just installed the foceiDur branch of `nlmixr`, and when trying to estimate a model, I got the error `function 'n1qn1_' not provided by package 'n1qn1'`.

I assume that I need this C version of n1qn1 to work with foceiDur.  Is that correct, or should I open an issue in `nlmixr` for that error?

**Comment by billdenney** [comment]
Does this package need a Makevars.in file with the line `PKG_LIBS = $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS)`?  (I tried adding this, but it didn't work for me.  I assume that there is more to it.)

**Comment by billdenney** [comment]
I made a push on top of yours that keeps the Makevars files.  Installation now works for me.

**Comment by mattfidler** [comment]
Ok.  I made some changes too.  It installs on windows for me.

**Comment by billdenney** [comment]
Can you take a look at my PR?  It fixes the deletion of Makevars, removes the .win variants of the files (they are not required), and fixes the tests.  Also, $(FLIBS) is suggested or required since BLAS is usually written in Fortran (according to one of the pages I found along my search and a Travis warning).

**Comment by mattfidler** [comment]
Thank you Bill for taking the time here.  We were working in parallel and got much of the same thing in the commit.

There were some differences: I kept the `.win` variant in case there was something that I needed to differently with windows.  I deleted the `cleanup` scripts since they were left-over from the fortran fork.  I added the `$(FLIBS)` and expanded the Makefile to include the files in case I have to have custom compilation for any of the files.

I saw you fixed the tests from my fork which I forgot to fix, so I will probably cherry pick that commit (if I can figure out how).
